jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/errorlist/ExecutionErrorListPresenter.java

```diff
@@ -0,0 +1,416 @@
+        return new ExecutionErrorSummary(
+                dataSetQueryHelper.getColumnStringValue(dataSet,
+                                                        COLUMN_ERROR_ID,
+                                                        index),
+                dataSetQueryHelper.getColumnStringValue(dataSet,
+                                                        COLUMN_ERROR_TYPE,
+                                                        index),
+                dataSetQueryHelper.getColumnStringValue(dataSet,
+                                                        COLUMN_DEPLOYMENT_ID,
+                                                        index),
+                dataSetQueryHelper.getColumnLongValue(dataSet,
+                                                      COLUMN_PROCESS_INST_ID,
+                                                      index),
+                dataSetQueryHelper.getColumnStringValue(dataSet,
+                                                        COLUMN_PROCESS_ID,
+                                                        index),
+                dataSetQueryHelper.getColumnLongValue(dataSet,
+                                                      COLUMN_ACTIVITY_ID,
+                                                      index),
+                dataSetQueryHelper.getColumnStringValue(dataSet,
+                                                        COLUMN_ACTIVITY_NAME,
+                                                        index),
+                dataSetQueryHelper.getColumnLongValue(dataSet,
+                                                      COLUMN_JOB_ID,
+                                                      index),
+                dataSetQueryHelper.getColumnStringValue(dataSet,
+                                                        COLUMN_ERROR_MSG,
+                                                        index),
+                dataSetQueryHelper.getColumnBooleanValue(dataSet,
+                                                         COLUMN_ERROR_ACK,
+                                                         index),
+                dataSetQueryHelper.getColumnStringValue(dataSet,
+                                                        COLUMN_ERROR_ACK_BY,
+                                                        index),
+                dataSetQueryHelper.getColumnDateValue(dataSet,
+                                                      COLUMN_ERROR_ACK_AT,
+                                                      index),
+                dataSetQueryHelper.getColumnDateValue(dataSet,
+                                                      COLUMN_ERROR_DATE,
+                                                      index)
+        );
+    }
+
+    public void OnExecErrorChanged(@Observes ExecErrorChangedEvent errorChangedEvent) {
+        refreshGrid();
+    }
+
+    @Override
+    @OnOpen
+    public void onOpen() {
+        this.textSearchStr = place.getParameter(COLUMN_ERROR_ID,
```

Comment:
This search can also be removed as we will review navigation across perspectives using the new search tab.